packages/reporter/package.json

```diff
@@ -19,7 +19,9 @@
     "clean-deps": "rm -rf node_modules",
     "pretest": "npm run check-deps-pre",
     "test": "node ./scripts/test.js",
-    "lint": "bin-up eslint --fix lib/*.js scripts/*.js src/*.js* src/**/*.js*"
+    "lint": "bin-up eslint --fix lib/*.js scripts/*.js src/*.js* src/**/*.js*",
+    "cypress:open": "node ../../scripts/cypress open --project .",
```

Comment:
You forgot to delete this?

Reply:
These were added intentionally to run the reporter Cypress tests.